cmake-modules/AzureCodeCoverage.cmake

```diff
@@ -55,6 +55,8 @@ function(add_gcovr_html)
             -r ${BASEDIR} ${EXCLUDE_ARGS}
             --object-directory=${PROJECT_BINARY_DIR}
             -o ${args_TARGET_NAME}/index.html
+            --exclude-unreachable-branches
```

Comment:
Wouldn't we want to get rid of unreachable branches in the code, rather than skip it for coverage?

Can you share a couple of concrete examples of these?

Reply:
This was an attempt to combat false positive noise.